MBBSEmu/CPU/CPUCore.cs

```diff
@@ -1157,6 +1163,44 @@ private void WriteToSource(ushort result)
             }
         }
 
+        [MethodImpl(OpcodeCompilerOptimizations)]
+        private void Op_Aam()
+        {
+            var al = Registers.AL;
+            var imm = _currentInstruction.Immediate8;
+
+            Registers.AH = (byte)(al / imm);
```

Comment:
Should we use `imm`? Intel Docs say it's always divided by `10`

![image](https://user-images.githubusercontent.com/1139047/112239245-85669e80-8c1c-11eb-87f7-c85f504adf38.png)